README.md

```diff
@@ -734,31 +734,43 @@ render status: :forbidden
   ```
 
 * <a name="find"></a>
-  Favor the use of `find` over `where`
-when you need to retrieve a single record by id.
+  Favor the use of `find` over `where.take!`, `find_by!`, and `find_by_id!`
```

Comment:
This reads like `find_by_id!` is a recommended method whereas it is listed as bad a few lines below. Can you clarify the copy a little?

Reply:
> This reads like `find_by_id!` is a recommended method whereas it is listed as bad a few lines below. Can you clarify the copy a little?

"Favor the use of A over B, C, and D" reads clearly (at least to me) that A is the only favored item, while B, C, and D are not, so I'm a little confused by your comment- what exactly are you suggesting needs to be clarified?

Reply:
But isn’t `find_by!` recommended over `find_by_id!`?

Reply:
No- when you need to retrieve a record by primary key id and raise `ActiveRecord::NotFound` when the record is not found, `find` is favored over _any_ of the bang-methods. So `find_by!` is not recommended over `find_by_id!` for this case because neither is recommended at all.

I still can't think of any clearer way to state this other than "Favor the use of `find` over `where.take!`, `find_by!`, and `find_by_id!`, when you need to retrieve a single record by primary key id and raise `ActiveRecord::RecordNotFound` when the record is not found." Feel free to offer an alternative if you have one.

Reply:
Got it. Sorry I misread, I initially thought you were discouraging the use of `find_by!`, but in this case you're specifically talking about primary key queries. 👍